src/layouts/Footer/components/Footer-Container/FooterContainer.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react'
+import NavLink from './components/NavLink/NavLink';
+import ScrollUpButton from './components/ScrollUpButton/ScrollUpButton';
+
+
+
+const FooterContainer: FC = () => {
+    
+    return (
+        <div className='bg-[#03111B] text-white h-96 mx-auto flex flex-col justify-center max-w-7xl'>
```

Comment:
```suggestion
        <div className='bg-[#03111B] text-white h-[96px] mx-auto flex justify-center max-w-7xl'>
```
Tailwind 默认height里面应该没有96这么大的，默认格式-x相当于 4X px，没有默认配置的可以用[]设置。需要用到flex col 才能实现吗？除非希望渲染方式是一列一列的情况，一般我们不需要使用 flex-col ，flex 默认排序方式就可以满足

Reply:
除了 flex-col 都resolved, flex col负责三排组件的居中